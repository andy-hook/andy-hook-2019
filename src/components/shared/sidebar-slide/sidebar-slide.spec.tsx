import React from "react"
import renderer from "react-test-renderer"

import SidebarSlide from "./sidebar-slide"

describe("<SidebarSlide />", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<SidebarSlide>Child text</SidebarSlide>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
