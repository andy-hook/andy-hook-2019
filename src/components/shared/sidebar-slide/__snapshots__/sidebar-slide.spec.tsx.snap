// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SidebarSlide /> renders correctly 1`] = `
<div>
  <div
    className="sc-dnqmqq hNHpNn is-closed"
  >
    Child text
  </div>
</div>
`;
